boards/DCM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -0,0 +1,87 @@
+#include "App_SharedSetPeriodicCanSignals.h"
+#include "App_SetPeriodicCanSignals.h"
+
+//This generates an error and doesn't build: ISO C forbids forward references to 'enum' types [-Werror=pedantic]
+// Why?
+STATIC_DEFINE_APP_SET_PERIODIC_CAN_SIGNALS_IN_RANGE_CHECK(DcmCanTxInterface)
+
+// TODO: Implement PID controller to maintain DC bus power at 80kW
+// #680
+void App_SetPeriodicCanSignals_TorqueRequests(const struct DcmWorld *world)
+{
+    float MAX_SAFE_TORQUE_REQUEST =
```

Comment:
Where is this number taken from?

Reply:
I asked Geoff. He said 21Nm is the max torque each motor can output.

Reply:
Okay sounds good. Make sure you declare this as a const and move them below CanInterfaces

Reply:
ok. What's the point of moving it below the can interfaces?

Reply:
Variables should be moved closer to where they are used